Userland/Libraries/LibPDF/Value.h

```diff
@@ -179,9 +178,9 @@ class Value {
         int m_as_int;
         u32 m_as_ref;
         float m_as_float;
-        Object* m_as_object;
     };
 
+    RefPtr<Object> m_as_object;
```

Comment:
Consider Variant?

Reply:
I did, but it seemed like a monstrous approach. Alright, I'll give it another shot.

Reply:
I'm not sure how big of a change it would be, I just saw a union member being moved out and went "Variant? Variant!".
if it's too annoying, we can totally live with +`sizeof(void*)` there